svir/dialogs/viewer_dock.py

```diff
@@ -607,6 +640,37 @@ def load_losses_by_asset_aggr(
 
         self.filter_losses_by_asset_aggr()
 
+    def load_avg_losses_stats_aggr(
+            self, calc_id, session, hostname, output_type):
+        composite_risk_model_attrs = extract_npz(
+            session, hostname, calc_id, 'composite_risk_model.attrs',
+            message_bar=self.iface.messageBar())
+        if composite_risk_model_attrs is None:
+            return
+        self._get_tags(session, hostname, calc_id, self.iface.messageBar(),
+                       with_star=True)
+        self.update_list_selected_edt()
+
+        loss_types = composite_risk_model_attrs['loss_types']
+        self.loss_type_cbx.blockSignals(True)
+        self.loss_type_cbx.clear()
+        self.loss_type_cbx.addItems(loss_types)
+        self.loss_type_cbx.blockSignals(False)
+
+        to_extract = 'agglosses/%s' % loss_types[0]
+        npz = extract_npz(session, hostname, calc_id, to_extract,
+                          message_bar=self.iface.messageBar())
+        if npz['stats'] == numpy.array(None):
+            self.stats = ['rlz-0']
```

Comment:
Perhaps a comment saying that this happens when there is a single realization is in order.